lib/user/lib/src/models/timetable/period.dart

```diff
@@ -169,12 +169,17 @@ class Periods {
           number: 1, startTime: Time(hour: 7), endTime: Time(hour: 8));
     } else {
       final lastPeriod = getPeriods()?.last;
-      final durationOfLastPeriod =
+      final minutesOfLastPeriod =
```

Comment:
Just renaming the variable because `durationOfLastPeriod` suggests that the type is `Duration`.